nh_eobs_mental_health/models/nh_clinical_patient_observation_ews.py

```diff
@@ -156,12 +156,22 @@ def is_refusal_in_effect(self, cr, uid, activity_id,
             column = 'first_activity_id'
             first_act_order = 'ASC'
         cr.execute(
-            'SELECT refused '
-            'FROM refused_ews_activities '
+            'SELECT refused.refused, '
+            'acts.date_terminated '
+            'FROM refused_ews_activities AS refused '
+            'RIGHT OUTER JOIN wb_activity_ranked AS acts '
+            'ON acts.id = refused.id '
+            'RIGHT OUTER JOIN nh_clinical_spell AS spell '
+            'ON spell.activity_id = refused.spell_activity_id '
+            'LEFT JOIN last_finished_pme AS pme ON pme.spell_id = spell.id '
             'WHERE {column} = {id} '
-            'ORDER BY spell_activity_id ASC, '
-            'first_activity_id {first_act_order}, '
-            'last_activity_id DESC '
+            'AND coalesce(acts.date_terminated >= spell.move_date, TRUE) '
+            'AND coalesce(acts.date_terminated >= '
+            'pme.activity_date_terminated, TRUE) '
+            'AND (spell.obs_stop <> TRUE OR spell.obs_stop IS NULL) '
+            'ORDER BY refused.spell_activity_id ASC, '
+            'refused.first_activity_id {first_act_order}, '
+            'refused.last_activity_id DESC '
```

Comment:
Was going to suggest indentation here but that's just preference and we need to agree on a style guide first. I figured the best thing to do would be to just get an SQL linter so I've raised [a tech debt issue](https://jupiter.bjss.com/browse/EOBS-862) for it.